vignettes/intro_to_allotalot.Rmd

```diff
@@ -177,66 +183,108 @@ iris_hd
 aq_hd
 ```
 
-Here we assigned three truncated data frames to three variables, no unpacking.
+Here we assigned three truncated data frames to three variables,
+and the data frames are intact.
 
-#### character strings
+### Unpacking nested values
```

Comment:
If this is another example I think we can start the line with `### Example: Unpacking ..` for continuity, see example section titles above.

If this is not another example section, I think we can use a level 2 title instead of a level 3 title. The subsequent non-example section has a level 2 title.